tests/integration/go_ethereum/common.py

```diff
@@ -20,12 +20,10 @@ def _check_web3_clientVersion(self, client_version):
 
 
 class GoEthereumEthModuleTest(EthModuleTest):
-    @pytest.mark.xfail(reason='eth_submitHashrate deprecated in 1.8.22 for ethash_submitHashRate')
     def test_eth_submit_hashrate(self, web3):
         # https://github.com/ethereum/go-ethereum/commit/51db5975cc5fb88db6a0dba1826b534fd4df29d7
         super().test_eth_submit_hashrate(web3)
 
-    @pytest.mark.xfail(reason='eth_submitHashrate deprecated in 1.8.22 for ethash_submitHashRate')
```

Comment:
If these couple of tests just work as expected now, we can delete them from this file. They're only listed here if we want to hijack the default behavior in some way, e.g., mark them as expected failures.

Reply:
Heard. Just pushed this up.